src/scanner/index.ts

```diff
@@ -35,6 +33,38 @@ export async function sendDeleteWorkloadRequest(workloadName: string, localWorkl
   await deleteWorkload(deletePayload);
 }
 
+export function getUniqueImages(workloadMetadata: IWorkload[]): IScanImage[] {
+  const uniqueImages: { [key: string]: IScanImage } = workloadMetadata.reduce((accum, meta) => {
+    logger.info({
+      workloadName: workloadMetadata[0].name,
+      name: meta.imageName,
+      id: meta.imageId
+    }, 'image metadata');
```

Comment:
Wouldn't this print super frequently?

Reply:
I thought of giving this logger day or two, just to have sample data. WDYT?

Reply:
and then remove it

Reply:
Sure!